README.md

```diff
@@ -117,11 +119,68 @@ yarn dev
 
 This will start both the NextJS app and a listener script monitoring your local blockchain for changes.
 
-## Gotchas
+# Deploying the dApp
+
+This section details the playbook for deploying the dApp.
+
+## Environments
+
+Commits to the following branches will automatically deploying to the associated Heroku environments:
+
+1. `stable` -> [Production](https://governance.ousd.com/claim)
+2. `staging` -> [Staging](https://ousd-governance-rinkeby.herokuapp.com/)
+
+The production environment references the mainnet contracts in this repo. The staging environment references contracts deployed to Rinkeby.
+
+Note: You shouldn't commit to `stable` directly. Only merge from `master` (where new features are merged via approved pull request).
+
+## Production Deployment Process
+
+1. Take a database backup before you start: `heroku pg:backups:capture --app=ousd-governance-production`. Note: You must be authorised with Heroku to run this command. Contact [Franck](mailto:franck@originprotocol.com) to be added if you're not already.
```

Comment:
I receive a lot of spam already. maybe don't include my email. just say contact Franck :)